lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -933,7 +933,7 @@ function dosomething_campaign_get_goals_times($node) {
   $timing = [
     "today" => new DateTime(),
     "start_date" => new DateTime($high_season[0]['value']),
-    "end_date" => new DateTime($high_season[0]['value2']),
+    "end_date" => date_modify(new DateTime($high_season[0]['value2']), '+23 hours +59 minutes +59 seconds'),
```

Comment:
curious why you switched from `setTime` to `datmodify`
https://github.com/DoSomething/phoenix/blob/beb2cbb520264f8b3abfbe8be6fc6ba6b13367ea/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module#L899